.eslintrc.json

```
@@ -0,0 +1,21 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "parser": "babel-eslint",
+    "parserOptions": {
+        "ecmaVersion": 12,
+        "sourceType": "module"
+    },
+    "plugins": ["svelte3"],
+    "extends": ["eslint:recommended"],
+    "overrides": [
+        {
+        "files": ["**/*.svelte"],
+        "processor": "svelte3/svelte3"
+        }
+    ],
+    "rules": {
+    }
```

we could add some rules that we'll have on EddieBot on this PR: https://github.com/EddieJaoudeCommunity/EddieBot/pull/287/files
```suggestion
    "rules": {
        "quotes": ["error", "single"],
        "indent": ["error", 2],
        "semi": ["error", "always"]
    }
```